src/main/java/org/jitsi/jicofo/stats/Statistics.java

```diff
@@ -18,4 +18,24 @@
      * Jicofo since it was started
      */
     public final AtomicInteger totalParticipants = new AtomicInteger(0);
+
+    /**
+     * The number of participants that were moved away from a failed bridge.
+     */
+    public final AtomicInteger totalParticipantsMoved = new AtomicInteger();
```

Comment:
Suggest these use `LongAdder`?  But probably all the stats should be updated at once.

Reply:
I don't think any of these are updated often enough for performance to be a concern.